### PR TITLE
SEO-176062 -Windowsforms-UG -Duplicate title

### PR DESCRIPTION
**Pull request by Edithabich**
HI @MallikaRK ,
Have updated UG titles please review.
Thanks.

Regards,
Edith Abich.

### CODE REVIEW

File: WindowsForms/skins/Overview.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: About Windows Forms skins control | Syncfusion
+title: Overview of Windows Forms skins control | Syncfusion
```
Comment (Yvone-Atieno) [inline WindowsForms/skins/Overview.md:3]: Overview of Windows Forms Skins Control | Syncfusion

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: WindowsForms/MultiColumn-ComboBox/Overview.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: About Windows Forms MultiColumnComboBox control | Syncfusion
+title: Overview of Windows Forms MultiColumnComboBox control | Syncfusion
```
[the remark this answers is not in the document]
Comment (Edithabich) [reply WindowsForms/MultiColumn-ComboBox/Overview.md]: resolved

File: WindowsForms/layoutmanagers/GridLayout/Overview.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: About Windows Forms LayoutManagers control | Syncfusion
+title: Overview of Windows Forms LayoutManagers control | Syncfusion
```
[the remark this answers is not in the document]
Comment (Edithabich) [reply WindowsForms/layoutmanagers/GridLayout/Overview.md]: resolved

File: WindowsForms/skins/Overview.md (hunk 1 above)
Comment (Edithabich) [reply WindowsForms/skins/Overview.md]: resolved

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Pull request closed without merging**